01_fib/alextmz/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	printfib(7)
+}
+
+func printfib(n int) {
+	// as per specs: intentionally skip 0
+	for i := 1; i <= n; i++ {
+		fmt.Fprintln(out, fibN(i))
+	}
```

Comment:
lab is solved, but not particularly run time efficient.
consider something like:
```go
	n1, n2 := 1, 1
	for i := 0; i < n; i++ {
		fmt.Println(n1)
		n1, n2 = n2, n1+n2
	}
```

Reply:
Indeed. I guess I tried to write simple code but even then I tried to separate the printing from the math which doesn't make much sense for _simple_ after all.